sources/packages/backend/apps/db-migrations/src/sql/Notifications/Create-notifications.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS notifications (
+    id SERIAL PRIMARY KEY,
+    -- Reference Columns
+    user_id INT REFERENCES users(id) ON DELETE CASCADE,
+    template_id VARCHAR(1000) NOT NULL,
```

Comment:
The template id will be a guid, right? I would say that it would be better mapped to a UUID. 1000 seems to be too much.